wolfcrypt/src/port/Renesas/renesas_tsip_aes.c

```diff
@@ -152,5 +242,252 @@ int wc_tsip_AesCbcDecrypt(struct Aes* aes, byte* out, const byte* in, word32 sz)
     return ret;
 }
 
+int wc_AesGcmEncrypt(
```

Comment:
Check indentation in this file (tabs should be 4 spaces). Please following our coding style: `if (`, `) {`, `else {` on new line, etc...

Reply:
Sorry for that.